vocabularies/UI.json

```diff
@@ -1750,6 +1750,12 @@
                 "@Core.Description": "Qualified name of an Action, Function, ActionImport or FunctionImport in scope"
             }
         },
+        "DataFieldWithValueHelp": {
```

Comment:
Since I was not involved so far I would like to give at least some feedback that I don't have a clue what this should be based on description and long description.

Typically a property has a ValueList annotation. But this seems to be a DataField
WithAction. Again I can't understand the use case based on what I see here. 

_A piece of data_ doesn't sound very nice to me

Reply:
Thinking about renaming `DataFieldWithValueHelp` to `DataFieldWithFunction`.

Reply:
👍🏻 This will help avoiding confusion with ValueList that many people call ValueHelp.

Reply:
`DataFieldWithValueFunction` In case you need another candidate